message_definitions/v1.0/messages.xml

```diff
@@ -338,7 +338,11 @@
       <field name="remote_noise" type="uint8"/>
     </message>
 
-    <!-- 40 is free -->
+    <message name="DCF" id="40">
+      <description>Telemetry message for monitoring the status of the Distributed Circular Formation</description>
+      <field name="table" type="int16[]"> 1. ID of the neighbor, 2. Theta of the neighbor (degrees x 100), 3. Desired inter-vehicle angle (degrees x 100), 4. Last time in ms we received a msg from the neighbor</field>
```

Comment:
do you know the size of the table in advance (e.g. the 4 elements described) or does it depend on the number of neighbors (n x 4 elements) ?